dbt_semantic_interfaces/transformations/default_granularity.py

```diff
@@ -49,11 +53,16 @@ def transform_model(semantic_manifest: PydanticSemanticManifest) -> PydanticSema
                     except AssertionError:
                         # This indicates the agg_time_dimension is misconfigured, which will fail elsewhere.
                         # Do nothing here to avoid disrupting the validation process.
+                        logger.warning(
+                            f"Agg time dimension '{agg_time_dimension_ref.element_name}' not found in model."
+                            "This should raise a validation error elsewhere."
```

Comment:
Where is it supposed to raise a validation error?

Reply:
https://github.com/dbt-labs/dbt-semantic-interfaces/blob/49a0347597b60fbec8d17771fa561c858f06b046/dbt_semantic_interfaces/validations/agg_time_dimension.py#L47-L48